src/redux/reducers/dataReducer.js

```diff
@@ -52,6 +53,14 @@ export default function(state = initialState, action) {
       return {
         ...state,
       };
+    case COMMENT_POST:
+      return {
+        ...state,
+        post: {
+          ...state.post,
+          commentList: [...state.post.commentList, action.payload],
```

Comment:
this could all be simplified using lodash's merge function:
```
return merge( { }, state, { commentList: [ action.payload ] })
```

Reply:
note: merge mutates the original parameter in addition to returning, hence the use of the empty object in the first param.

Reply:
fixed